src/App.tsx

```diff
@@ -1,7 +1,12 @@
 import React from 'react';
+import OrderPage from 'views/OrderPage';
```

Comment:
This import does not exist, OrderPage is not exported from views/OrderPage, it's styles are.